SVG4Everybody Failed connect to github.com:443

Since I cleaned up our `dependecies` and `devDependencies` hash we get the following error inside of Jenkins (I guess it is this proxy issue, that he can't access github).

Anyway there is already a npm package available for SVG4Everybody, so I will use this instead.

> npm ERR! git clone --template=/var/opt/jet/nodejs/npm-cache/_git-remotes/_templates --mirror https://github.com/jonathantneal/svg4everybody.git /var/opt/jet/nodejs/npm-cache/_git-remotes/https-github-com-jonathantneal-svg4everybody-git-6578e680: Cloning into bare repository '/var/opt/jet/nodejs/npm-cache/_git-remotes/https-github-com-jonathantneal-svg4everybody-git-6578e680'...
> npm ERR! git clone --template=/var/opt/jet/nodejs/npm-cache/_git-remotes/_templates --mirror https://github.com/jonathantneal/svg4everybody.git /var/opt/jet/nodejs/npm-cache/_git-remotes/https-github-com-jonathantneal-svg4everybody-git-6578e680: fatal: unable to access 'https://github.com/jonathantneal/svg4everybody.git/': Failed connect to github.com:443; Operation now in progress
> npm ERR! Linux 2.6.32-642.1.1.el6.x86_64
> npm ERR! argv "/opt/jet/nodejs/node-v4.3.1-linux-x64/bin/node" "/opt/jet/nodejs/node-js/bin/npm" "install"
> npm ERR! node v4.3.1
> npm ERR! npm  v2.14.12
> npm ERR! code 128
> 
> npm ERR! Command failed: git clone --template=/var/opt/jet/nodejs/npm-cache/_git-remotes/_templates --mirror https://github.com/jonathantneal/svg4everybody.git /var/opt/jet/nodejs/npm-cache/_git-remotes/https-github-com-jonathantneal-svg4everybody-git-6578e680
> npm ERR! Cloning into bare repository '/var/opt/jet/nodejs/npm-cache/_git-remotes/https-github-com-jonathantneal-svg4everybody-git-6578e680'...
> npm ERR! fatal: unable to access 'https://github.com/jonathantneal/svg4everybody.git/': Failed connect to github.com:443; Operation now in progress
> npm ERR! 
> npm ERR! 
> npm ERR! If you need help, you may report this error at:
> npm ERR!     <https://github.com/npm/npm/issues>
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     /var/opt/jet/jenkins-ch-prod-node-3-lxfjet21/workspace/aem_core-it_dev_nightly/frontend-lib/src/npm-debug.log

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "baconjs": "^0.7.73",
     "jquery": "^2.1.3",
     "moment": "^2.9.0",
-    "svg4everybody": "git+https://github.com/jonathantneal/svg4everybody"
+    "svg4everybody": "^2.1.0"
   },
   "devDependencies": {
     "autoprefixer": "^6.3.7",

